src/vorta/views/repo_add_dialog.py

```diff
@@ -242,10 +263,10 @@ def run(self):
             params = BorgInfoRepoJob.prepare(self.values)
             if params['ok']:
                 self.saveButton.setEnabled(False)
-                thread = BorgInfoRepoJob(params['cmd'], params)
-                thread.updated.connect(self._set_status)
-                thread.result.connect(self.run_result)
-                self.thread = thread  # Needs to be connected to self for tests to work.
-                self.thread.run()
+                job = BorgInfoRepoJob(params['cmd'], params, params['repo_id'])
```

Comment:
here I was confused, I was actually unable to test this, this would just get the `Profile` from `info_repo` 🤔

Reply:
This is right.